projects/core/src/cart/occ/cart.service.ts

```diff
@@ -29,17 +29,14 @@ const DETAILS_PARAMS =
 
 @Injectable()
 export class OccCartService {
-  constructor(protected http: HttpClient, protected config: OccConfig) {}
+  constructor(
+    protected http: HttpClient,
+    private occEndpoints: OccEndpointsService
+  ) {}
 
   protected getCartEndpoint(userId: string) {
```

Comment:
can we add `:void`

Reply:
It actually should be a `string`, thanks, fixed.